.gitignore

```diff
@@ -1,2 +1,3 @@
 *.exe
-vendor
\ No newline at end of file
+vendor
+.DS_Store
```

Comment:
Where is `desktop.ini`? 🥲🥲🥲

Reply:
It will be ignored in next config.
Also, who uses windows for developing? : D